web/packages/teleterm/src/ui/services/usage/setUpUsageReporting.ts

```diff
@@ -29,6 +29,11 @@ export async function setUpUsageReporting(
     return;
   }
 
+  if (configService.getConfigError()?.source === 'file-loading') {
+    // do not show the dialog, response cannot be saved to the file
+    return;
```

Comment:
This makes sense to me, the main argument being what you mentioned – if the response cannot be saved then it doesn't make sense to ask the question anyway.